dgl_ptm/dgl_ptm/model/initialize_model.py

```diff
@@ -105,19 +106,27 @@ def __init__(self,*, model_identifier=None, restart=False, savestate=None):
 
     def set_model_parameters(self,*,parameterFilePath=None, default=True, **kwargs):
         """
-        Load (TODO) or set model parameters
+        Load or set model parameters
 
-        :param parameterFlePath: optional, path to restore file (TODO)
+        :param parameterFlePath: optional, path to parameter file
         :param default: Specify whether default values should be used (True;default)
-        :param **kwargs: flexible passing of mode parameters. Only those supported by the model are accpted.
+        :param **kwargs: flexible passing of mode parameters. Only those supported by the model are accepted.
                          If parameters are passed, non-specifed parameters will be set with defaults.
 
         """
+        modelpars = self.__dict__.keys()
         if parameterFilePath != None:
```

Comment:
This looks fine. The only ting that stil doesn't realy work well is model naming. If aparameter file is used it might be good to read the indentifier from file and/or also retain the parameter file name. I'll raise this as a separate issue